src/vue/pages/BusinessViewer.vue

```diff
@@ -214,6 +214,10 @@ export default {
     justify-content: space-between;
   }
 
+  .app__button-raised {
```

Comment:
You can't redefine this style. This global app style for all buttons. After the user will visit this screen, all buttons with this class get margin-top. If you want to supplement style add new class. But why you add this margin?
Remove this.

Reply:
removed